scripts/codeCoverage.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+LAUNCH_DIR=`pwd`
+
+#  uncoment to enabled source blacklist: exmple --> SOURCE_BLACKLIST=".*ProbeTransmogrifier.cpp.*"
+HEADER_WHITELIST=
```

Comment:
OK. I will add that

*Contact Information*
*Kjell Hedström*
*Mobile:  +1 - 617.378.1915*

*E-mail: Hedstrom@KjellKod.cc*




On Tue, Jan 12, 2016 at 11:05 AM, john-gress <notifications@github.com>
wrote:

> In scripts/codeCoverage.sh
> <https://github.com/weberr13/FileIO/pull/15#discussion_r49489929>:
>
> > @@ -0,0 +1,73 @@
> > +#!/bin/bash
> > +LAUNCH_DIR=`pwd`
> > +
> > +#  uncoment to enabled source blacklist: exmple --> SOURCE_BLACKLIST=".*ProbeTransmogrifier.cpp.*"
> > +HEADER_WHITELIST=
>
> I think you need a comment to explain why one would want to add a file to
> the white list.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/weberr13/FileIO/pull/15/files#r49489929>.
>